### ISSUE TITLE
Wrong import statement affecting `LinearWarmupCosineAnnealingLR` (and probably more code)

### ISSUE THREAD

**Issue by danielgafni** [Lightning-Universe/lightning-bolts#977]
## 🐛 Bug

Can't import `LinearWarmupCosineAnnealingLR`

### To Reproduce
```python
from pl_bolts.optimizers.lr_scheduler import LinearWarmupCosineAnnealingLR
```

```
----> 1 from pl_bolts.optimizers.lr_scheduler import LinearWarmupCosineAnnealingLR
      3 class EventsTransformer(BaseEventsModel):
      4     def __init__(
      5         self,
      6         features: Sequence[str] | None = None,
   (...)
     17         dropout: float = 0.2
     18     ):

File ~/.cache/pypoetry/virtualenvs/ar-resp-vArrGwjy-py3.10/lib/python3.10/site-packages/pl_bolts/__init__.py:11
      8 _PROJECT_ROOT = os.path.dirname(_PACKAGE_ROOT)
      9 _HTTPS_AWS_HUB = "https://pl-bolts-weights.s3.us-east-2.amazonaws.com/"
---> 11 from pl_bolts import (  # noqa: E402
     12     callbacks,
     13     datamodules,
     14     datasets,
     15     losses,
     16     metrics,
     17     models,
     18     optimizers,
     19     transforms,
     20     utils,
     21 )
     23 __all__ = [
     24     "callbacks",
     25     "datamodules",
   (...)
     32     "utils",
     33 ]

File ~/.cache/pypoetry/virtualenvs/ar-resp-vArrGwjy-py3.10/lib/python3.10/site-packages/pl_bolts/callbacks/__init__.py:3
      1 """Collection of PyTorchLightning callbacks."""
      2 from pl_bolts.callbacks.byol_updates import BYOLMAWeightUpdate
----> 3 from pl_bolts.callbacks.data_monitor import ModuleDataMonitor, TrainingDataMonitor
      4 from pl_bolts.callbacks.printing import PrintTableMetricsCallback
      5 from pl_bolts.callbacks.sparseml import SparseMLCallback

File ~/.cache/pypoetry/virtualenvs/ar-resp-vArrGwjy-py3.10/lib/python3.10/site-packages/pl_bolts/callbacks/data_monitor.py:6
      4 import torch
      5 from pytorch_lightning import Callback, LightningModule, Trainer
----> 6 from pytorch_lightning.loggers import LightningLoggerBase, TensorBoardLogger, WandbLogger
      7 from pytorch_lightning.utilities import rank_zero_warn
      8 from pytorch_lightning.utilities.apply_func import apply_to_collection

ImportError: cannot import name 'LightningLoggerBase' from 'pytorch_lightning.loggers'
```
### Expected behavior

The code should work

### Environment

- Lightning-Bolts: 0.6.0.post1
- PyTorch-Lightning: 1.9.0
- Python version: 3.10

**Comment by JohnKilbride** [comment]
I ran into this same issue! A work around I found was to use LinearWarmupCosineAnnealingLR from the Flash module.

**Comment by a-r-j** [comment]
I'm running into the same issue. 

torch: `2.0.0`
lightning: `2.0.0`
lightning-bolts: `0.6.0.post1`

**Comment by Deekshith458** [comment]
Did someone find the fix yet?

**Comment by Borda** [comment]
shall be fixed in #973
